helm/values.yaml

```diff
@@ -26,7 +26,10 @@ fullnameOverride: ""
 envVars:
   shard: {}
   core: {}
-  frontend: {}
```

Comment:
If we mount the output of the `env.sh` script inside the container instead of generating it in the runtime, we wouldn't need this variable anymore.

Reply:
I totally agree with this as you explained earlier that this might increase the chance of mistakes by the users.